tests/composite_pk/tests.py

```diff
@@ -149,14 +149,14 @@ def test_in_bulk(self):
 
     def test_in_bulk_batching(self):
         Comment.objects.all().delete()
-        batching_required = connection.features.max_query_params is not None
-        expected_queries = 2 if batching_required else 1
+        connection.features.__dict__.pop("max_query_params", None)
```

Comment:
Can you tell me a little bit about this change? Was there a test isolation problem? What is the state of `connection.features` after this test?